docs/pages/billing/plans.mdx

```diff
@@ -0,0 +1,94 @@
+---
+title: Subscriptions, plans, and add-ons
+description: In-depth guide on available Expo Application Services (EAS) plans and how they work, usage-based pricing and add-ons.
+---
+
+import { BoxLink } from '~/ui/components/BoxLink';
+import { CreditCard02Icon } from '@expo/styleguide-icons';
+import ImageSpotlight from '~/components/plugins/ImageSpotlight';
+
+[Expo Application Services (EAS)](/eas/) offers [free access](https://expo.dev/eas/fair-use#commercial-usage) to a limited quantity of low-priority builds on [EAS Build](/build/introduction/) and free updates with [EAS Update](/eas-update/introduction/). These limits reset monthly.
+
+Beyond the Free plan, there are different subscription plans to cater to various customers and their needs. Each plan offers credits to enable high-priority builds for EAS Build and broader access to EAS Update through more monthly updated users and extra bandwidth. We also offer add-ons that complement subscriptions and enable opt-in features to amplify customer needs.
```

Comment:
```suggestion
Beyond the Free plan, there are different subscription plans to cater to various customers and their needs. Each plan offers credits to enable priority builds for EAS Build and broader access to EAS Update through more monthly updated users and extra bandwidth. We also offer add-ons that complement subscriptions and enable opt-in features to amplify customer needs.
```
Matching the pricing page verbiage ... just didn't want to imply there's a 3rd "higher" priority queue.